### PR TITLE
increased prow limits

### LINKED ISSUE #99: Pre commit - Pod pending issue
Increase the prow resource limits to fix this issue.

### PR DESCRIPTION
**Pull request by antter**
Fixes #99

### CODE REVIEW

**Comment by antter** [comment]
@aakankshaduggal What is this prow error message?

File: .prow.yaml [hunk 1]
```diff
@@ -16,5 +16,5 @@ presubmits:
               memory: "500Mi"
               cpu: "300m"
             limits:
-              memory: "1Gi"
-              cpu: "500m"
+              memory: "2Gi"
+              cpu: "1Gi"
```
Comment (aakankshaduggal) [inline .prow.yaml:20]: I think cpu value is wrong here. Make it 500m and try again.

File: .prow.yaml [hunk 2]
```diff
@@ -16,5 +16,5 @@ presubmits:
               memory: "500Mi"
               cpu: "300m"
             limits:
-              memory: "1Gi"
-              cpu: "500m"
+              memory: "2Gi"
+              cpu: "500Mi"
```
Comment (aakankshaduggal) [inline .prow.yaml:20]: 500m

**Comment by aakankshaduggal** [comment]
Hey @harshad16, does this look okay? 
We were getting pre-commit issues as mentioned in #99. Please let us know if this is the right way to go.

**Comment by harshad16** [comment]
> does this look okay? We were getting pre-commit issues as mentioned in #99. Please let us know if this is the right way to go.

yes it does
please can you also update this file:
https://github.com/aicoe-aiops/github-labeler/blob/master/.pre-commit-config.yaml
based on this comment 
https://github.com/thoth-station/thoth-application/issues/2111#issuecomment-957617546

**Review by harshad16 — COMMENTED** [review]
lgtm
thanks :100:

**Review by aakankshaduggal — COMMENTED** [review]
/approve

**Pull request merged**